config/routes.rb

```diff
@@ -216,10 +216,6 @@
     end
   end
 
-  resources :config_groups, :except => [:show] do
-    get 'auto_complete_search', :on => :collection
-  end
```

Comment:
This should be definitelly part of the UI drop and we should drop https://github.com/theforeman/foreman/blob/324bafdb625b06fe939530ef77534170be855447/config/routes/api/v2.rb#L254 here instead.